pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -244,19 +251,20 @@ func (b *ConsoleServerCLIConfigBuilder) CopiedCSVsDisabled(copiedCSVsDisabled bo
 
 func (b *ConsoleServerCLIConfigBuilder) Config() Config {
 	return Config{
-		Kind:           "ConsoleConfig",
-		APIVersion:     "console.openshift.io/v1",
-		Auth:           b.auth(),
-		Session:        b.session(),
-		ClusterInfo:    b.clusterInfo(),
-		Customization:  b.customization(),
-		ServingInfo:    b.servingInfo(),
-		Providers:      b.providers(),
-		MonitoringInfo: b.monitoringInfo(),
-		Plugins:        b.plugins(),
-		I18nNamespaces: b.i18nNamespaces(),
-		Proxy:          b.proxy(),
-		Telemetry:      b.telemetry,
+		Kind:                  "ConsoleConfig",
+		APIVersion:            "console.openshift.io/v1",
+		Auth:                  b.auth(),
+		Session:               b.session(),
+		ClusterInfo:           b.clusterInfo(),
+		Customization:         b.customization(),
+		ServingInfo:           b.servingInfo(),
+		Providers:             b.providers(),
+		MonitoringInfo:        b.monitoringInfo(),
+		Plugins:               b.plugins(),
+		I18nNamespaces:        b.i18nNamespaces(),
+		Proxy:                 b.proxy(),
+		ContentSecurityPolicy: b.contentSecurityPolicy(),
```

Comment:
@TheRealJon eventually we should unify the plugins related field (`Plugins`, `I18Namespaces`, `Proxy` and `ContentSecurityPolicy`) under a single field eg. `PluginInfo`, for consistency.